### PR TITLE
Attempting to update to Fable 1.2 and dotnet core sdk 2.0

### LINKED ISSUE #16: Update to latest Fable
* update to latest Fable
* .fsproj without FSharp.NET.Sdk, 
* Paket storage:none for Fable dependencies

### PR DESCRIPTION
**Pull request by TheAngryByrd**
Closes #16 

However, I'm currently getting

```
System.Exception: dotnet command failed on fable webpack --port free
  at Fake.DotNetCli+RunCommand@78-3.Invoke (System.String message) [0x00001] in <597b0a4fccf1c534a74503834f0a7b59>:0 
  at Microsoft.FSharp.Core.PrintfImpl+StringPrintfEnv`1[TResult].Finalize () [0x00012] in <5893d081904cf4daa745038381d09358>:0 
  at Microsoft.FSharp.Core.PrintfImpl+Final1@224[TState,TResidue,TResult,A].Invoke (Microsoft.FSharp.Core.FSharpFunc`2[T,TResult] env, A a) [0x00038] in <5893d081904cf4daa745038381d09358>:0 
  at Microsoft.FSharp.Core.OptimizedClosures+Invoke@3253[T2,TResult,T1].Invoke (T2 u) [0x00001] in <5893d081904cf4daa745038381d09358>:0 
  at Fake.DotNetCli.RunCommand (Microsoft.FSharp.Core.FSharpFunc`2[T,TResult] setCommandParams, System.String args) [0x00055] in <597b0a4fccf1c534a74503834f0a7b59>:0 
  at FSI_0005.Build+clo@103-14.Invoke (System.String proj) [0x0000e] in <d0ee8ca688eb46e49b491023943d5b46>:0 
  at Microsoft.FSharp.Collections.SeqModule.Iterate[T] (Microsoft.FSharp.Core.FSharpFunc`2[T,TResult] action, System.Collections.Generic.IEnumerable`1[T] source) [0x0002d] in <5893d081904cf4daa745038381d09358>:0 
  at FSI_0005.Build+clo@101-13.Invoke (Microsoft.FSharp.Core.Unit _arg1) [0x00014] in <d0ee8ca688eb46e49b491023943d5b46>:0 
  at Fake.TargetHelper+targetFromTemplate@209-1[a].Invoke (Microsoft.FSharp.Core.Unit unitVar0) [0x00001] in <597b0a4fccf1c534a74503834f0a7b59>:0 
  at Fake.TargetHelper.runSingleTarget (Fake.TargetHelper+TargetTemplate`1[a] target) [0x0004b] in <597b0a4fccf1c534a74503834f0a7b59>:0 
```

Not sure why paket isn't restoring the clitool correctly.

### CODE REVIEW

**Comment by TheAngryByrd** [comment]
@MangelMaxime have you seen paket fail to install the clitools before? Or am I missing something obvious.

File: build.fsx [hunk 1]
```diff
@@ -9,23 +9,11 @@ open System
 let release = LoadReleaseNotes "RELEASE_NOTES.md"
 let srcGlob = "*.csproj"
 
-
 Target "Clean" (fun _ ->
     [ "obj" ;"dist"]
     |> CleanDirs
     )
 
-Target "DotnetRestore" (fun _ ->
```
Comment (MangelMaxime) [inline build.fsx]: Why remove this target ? It's the one make restoring the deps for the projects

File: build.sh [hunk 2]
```diff
@@ -36,10 +36,7 @@ run() {
 
 
 
-run $PAKET_EXE restore
-
-[ ! -e build.fsx ] && run $PAKET_EXE update
-[ ! -e build.fsx ] && run $FAKE_EXE init.fsx
+run $PAKET_EXE restore --group build
```
Comment (MangelMaxime) [inline build.sh]: Why restore only the build group ? You will miss others deps no ? Like `dotnet-fable`

File: Content/src/Fable.MyLib.fsproj [hunk 3]
```diff
@@ -1,16 +1,13 @@
-<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Version>0.0.0</Version>
-    <TargetFramework>netstandard1.6</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```
Comment (MangelMaxime) [inline Content/src/Fable.MyLib.fsproj]: It's better for libs to target the lower framework.
As long as we don't use any specific feature of netstandard2.0.

Here you will end up to force users to update their project even if it's not really needed.

File: Content/paket.dependencies [hunk 4]
```diff
@@ -1,7 +1,7 @@
 source https://www.nuget.org/api/v2
 
 nuget FSharp.Core  redirects:force, content:none
-nuget Fable.Core
+nuget Fable.Core content:none
```
Comment (MangelMaxime) [inline Content/paket.dependencies]: You should take this paket.dependencies as sample IMO

https://github.com/fable-compiler/fable-templates/blob/master/simple/Content/paket.dependencies

**Comment by MangelMaxime** [comment]
Today, I'm a bit in hurry (delivery at work).

If you didn't manage to fix your build, I will do it this evening :)

**Comment by TheAngryByrd** [comment]
Awesome thanks for the review!

File: build.sh (hunk 2 above)
Comment (TheAngryByrd) [reply build.sh]: This is for the top level one. Also seems like paket wasn't restoring all groups for whatever reason.

File: build.fsx (hunk 1 above)
Comment (TheAngryByrd) [reply build.fsx]: Didn't think it was necessary for the top level project anymore.

File: Content/src/Fable.MyLib.fsproj (hunk 3 above)
Comment (TheAngryByrd) [reply Content/src/Fable.MyLib.fsproj]: Thanks for the clarification! I'm still getting my head around .net core 2.0 and version number overload.

**Comment by TheAngryByrd** [comment]
Yeah If I add 

```
<ItemGroup>
 <DotNetCliToolReference Include="dotnet-fable" Version="1.2.0" />
</ItemGroup>
```

Directly to the tests fsproj file it gets restored properly but without paket doesn't seem to be restoring it :(

**Comment by TheAngryByrd** [comment]
Ok yeah i'm gonna reduce cleanups because i got it working in another branch.

**Pull request merged**